app/controllers/concerns/breadcrumbable.rb

```diff
@@ -20,7 +20,7 @@ def current_page
       route.name
     end
 
-    routes.flatten.last.name
+    routes.first.last.name
```

Comment:
:exploding_head:

Reply:
i can't even... i'm so far down the rabbit hole of nested rails application router.. it's unreal.